pkgs/development/web/flyctl/default.nix

```
@@ -32,9 +32,14 @@ buildGoModule rec {
   '';
 
   postCheck = ''
-    go test ./... -ldflags="-X 'github.com/superfly/flyctl/internal/buildinfo.buildDate=1970-01-01T00:00:00+0000'"
+    go test ./... -ldflags="-X 'github.com/superfly/flyctl/internal/buildinfo.buildDate=1970-01-01T00:00:00Z'"
   '';
 
+  passthru.tests.version = testers.testVersion {
+    package = flyctl;
```

```suggestion
    package = flyctl;
    version = "v${flyctl.version}";
```
does it work like this?

---

that works

```
[nix-shell:~/.cache/nixpkgs-review/pr-172428-2]$ nix-build nixpkgs/ -A flyctl.tests
this derivation will be built:
  /nix/store/l5sbn1jyb88pqgzmnpj960kxy2z9hw2g-flyctl-0.0.323-test-version.drv
building '/nix/store/l5sbn1jyb88pqgzmnpj960kxy2z9hw2g-flyctl-0.0.323-test-version.drv'...
flyctl v0.0.323 linux/amd64 Commit: v0.0.323 BuildDate: 1970-01-01T00:00:00Z
/nix/store/knwy3jq2d906dbp5mlc36x3qyj45ai4y-flyctl-0.0.323-test-version
```

`version` is not known in `passthru.tests` context from the package

---

maybe i'm wrong. it works also with `version`

@Artturin  what is more correct?